src/stylesheets/elements/typography/_field.scss

```diff
@@ -15,11 +15,19 @@
   .sds-field__name {
     @include u-display("inline-block");
     @include u-margin-right("2px");
+    // class used to vertically align field name to tag text
+    &.has-usa-tag{
```

Comment:
can we rename this to just .usa-tag? the `has` in there seems a bit unusual as a css name

Reply:
.usa-tag already exists. This class is meant to help vertically align the field name when next to a .usa-tag.

Reply:
Approved - it's a fairly minor thing - normally I see many variables start with has/is to indicate boolean, but the standards don't seem to carry over with class names in css

Reply:
Yeah I agree it is a bit awkward. Just trying to find a way to connect this class to .usa-tag since it really is only used when aligning to .usa-tag.